dev-util/codeblocks/codeblocks-20.03.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit autotools wxwidgets xdg
+
+DESCRIPTION="The open source, cross platform, free C, C++ and Fortran IDE"
+HOMEPAGE="http://www.codeblocks.org/"
```

Comment:
https://codeblocks.org/

Reply:
O_o
Sorry, obviously I copied from old ebuild of local overlay before edit it